[BUG] Build broken on Databricks 14.3 after #11996

**Describe the bug**
Build succeed without #11996 
and fails with it

```
$ ./jenkins/databricks/build.sh
[INFO] Compiling 45 Scala sources to /home/ubuntu/spark-rapids/delta-lake/delta-spark350db143/target/spark350db143/classes...
[ERROR] /home/ubuntu/spark-rapids/delta-lake/common/src/main/databricks/scala/org/apache/spark/sql/rapids/delta/GpuOptimizeWriteExchangeExec.scala:91: not enough arguments for constructor GpuColumnarBatchSerializer: (metrics: Map[String,com.nvidia.spark.rapids.GpuMetric], dataTypes: Array[org.apache.spark.sql.types.DataType], useKudo: Boolean, kudoMeasureBufferCopy: Boolean)com.nvidia.spark.rapids.GpuColumnarBatchSerializer.
Unspecified value parameter kudoMeasureBufferCopy.[ERROR]     new GpuColumnarBatchSerializer(allMetrics,
```

--- a/delta-lake/common/src/main/databricks/scala/org/apache/spark/sql/rapids/delta/GpuOptimizeWriteExchangeExec.scala
+++ b/delta-lake/common/src/main/databricks/scala/org/apache/spark/sql/rapids/delta/GpuOptimizeWriteExchangeExec.scala
@@ -90,7 +90,8 @@ case class GpuOptimizeWriteExchangeExec(
   private lazy val serializer: Serializer =
     new GpuColumnarBatchSerializer(allMetrics,
       child.output.map(_.dataType).toArray,
-      RapidsConf.SHUFFLE_KUDO_SERIALIZER_ENABLED.get(child.conf))
+      RapidsConf.SHUFFLE_KUDO_SERIALIZER_ENABLED.get(child.conf),
+      RapidsConf.SHUFFLE_KUDO_SERIALIZER_MEASURE_BUFFER_COPY_ENABLED.get(child.conf))
 
   @transient lazy val inputRDD: RDD[ColumnarBatch] = child.executeColumnar()
 

--- a/delta-lake/common/src/main/databricks/scala/org/apache/spark/sql/rapids/delta/GpuOptimizeWriteExchangeExec.scala
+++ b/delta-lake/common/src/main/databricks/scala/org/apache/spark/sql/rapids/delta/GpuOptimizeWriteExchangeExec.scala
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023-2024, NVIDIA CORPORATION.
+ * Copyright (c) 2023-2025, NVIDIA CORPORATION.
  *
  * This file was derived from OptimizeWriteExchange.scala
  * in the Delta Lake project at https://github.com/delta-io/delta

